wp-content/plugins/core/src/Theme/Config/Image_Sizes.php

```diff
@@ -17,6 +17,11 @@ class Image_Sizes {
 	public const FOUR_THREE_LARGE       = 'four-three-large';
 	public const SOCIAL_SHARE_OPENGRAPH = 'social-share-opengraph';
 	public const SOCIAL_SHARE_TWITTER   = 'social-share-twitter';
+	public const THREE_TWO_XSMALL       = 'three-two-xsmall';
+	public const THREE_TWO_SMALL        = 'three-two-small';
+	public const THREE_TWO              = 'three-two';
+	public const THREE_TWO_LARGE        = 'three-two-large';
+	public const THREE_TWO_XLARGE       = 'three-two-xlarge';
```

Comment:
Hmmm... Let's chat this through with design. I don't know if I'm keen on adding more image sizes to SquareOne by default. Each set of sizes we define adds additional intermediate sizes that have to be generated when an image is added. This creates a ton of overhead. 

Matt CR or @kyleunzicker , would it be possible to use one or two set of images sizes (aspect ratos) throughout the defaults in SquareOne? Of course we can adjust ARs for each project. I'm just hoping to avoid creating ~20+ sizes on ever new project.  Thoughts?